scripts/feature-flags.js

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+const ajv = require('ajv')();
+const Promise = require('../promise');
+const redis = require('../redis')({
+  enabled: true,
+  host: process.env.REDIS_HOST || '127.0.0.1',
+  port: process.env.REDIS_PORT || 6379,
+  prefix: 'featureFlags:',
+  maxConnections: 2,
+  maxPending: 1,
+  minConnections: 1,
+}, {
+  error () {},
+  info () {},
+  warn () {}
+});
+const schema = require('../feature-flags/schema.json');
+const validate = ajv.compile(schema);
+
+const COMMANDS = {
```

Comment:
What are your thoughts on pulling these commands into `feature-flags/index.js` and then using this script to interface into that library?

Reply:
`feature-flags/index.js` should only `get`, we never want it to write data to Redis. Excluding the write methods from that interface means it's impossible for a caller to do that accidentally (even though the probability of that seems remote).

Reply:
This seems like a reasonable protection to me, plus you have full integration tests below for the other commands.